src/components/application_manager/include/application_manager/request_controller_impl.h

```diff
@@ -47,173 +47,75 @@
 #include "interfaces/HMI_API.h"
 #include "interfaces/MOBILE_API.h"
 
+#include "application_manager/request_controller.h"
 #include "application_manager/request_controller_settings.h"
 #include "application_manager/request_info.h"
+#include "application_manager/request_timeout_handler_impl.h"
 #include "application_manager/request_tracker.h"
 
 namespace application_manager {
 
 namespace request_controller {
 
-/**
- * @brief RequestController class is used to control currently active mobile
- * requests.
- */
-class RequestController {
+class RequestControllerImpl : public RequestController {
```

Comment:
Please add all the deleted comments as applicable.

Reply:
@atiwari9 please note that `RequestControllerImpl` is now just a class containing public functions overridden from the interface class. We don't need to duplicate the description of such functions in the impl classes according to our coding style:
https://google.github.io/styleguide/cppguide.html#Function_Comments

> When documenting function overrides, focus on the specifics of the override itself, rather than repeating the comment from the overridden function. In many of these cases, the override needs no additional documentation and thus no comment is required.

Reply:
@AKalinich-Luxoft , @ychernysheva  - Please let me know when the PR is ready to be reviewed after above mentioned changes.